compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -345,6 +336,40 @@ static ConcreteMmaLayout getConcreteMFMALayout(MLIRContext *context,
   return concreteLayout;
 }
 
+//===----------------------------------------------------------------------===//
+// MmaInterface Attribute Helper Functions
+//===----------------------------------------------------------------------===//
+
+MMASingleSubgroupLayout getASingleSubgroupLayout(MmaInterfaceAttr mmaKind) {
+  if (auto mmaAttr = llvm::dyn_cast<MMAAttr>(mmaKind)) {
+    return mmaAttr.getASingleSubgroupLayout();
+  } else if (auto vmmaAttr = llvm::dyn_cast<VirtualMMAAttr>(mmaKind)) {
```

Comment:
nit: no else after return: https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Reply:
haven't seen this doc, very cool! will do this, thanks!